epictrack-api/src/api/services/staff_elevated_role.py

```diff
@@ -32,41 +32,45 @@ def find_all_active(cls):
         return StaffElevatedRoleModel.find_all_active_roles()
 
     @classmethod
-    def find_by_id(cls, identifier):
+    def find_by_id(cls, identifier, is_active=True):
         """Find staff elevated role by ID"""
         current_app.logger.debug("find staff elevated role by ID")
-        staff_elevated_role = StaffElevatedRoleModel.find_by_id(identifier)
+        staff_elevated_role = StaffElevatedRoleModel.find_by_id(identifier, include_inactive=not is_active)
         if not staff_elevated_role:
             raise ResourceNotFoundError(f"Staff Elevated Role with id '{identifier}' not found")
         return staff_elevated_role
 
     @classmethod
-    def find_by_staff_id(cls, staff_id):
+    def find_by_staff_id(cls, staff_id, is_active=True):
         """Find staff elevated roles by staff ID"""
         current_app.logger.debug("find staff elevated roles by staff_ID")
-        staff_elevated_role = StaffElevatedRoleModel.find_by_staff_id(staff_id)
+        staff_elevated_role = StaffElevatedRoleModel.find_by_staff_id(staff_id, include_inactive=not is_active)
         if staff_elevated_role:
+            current_app.logger.info(f"Found staff elevated role by staff_ID: {staff_elevated_role}")
```

Comment:
:thinking:

Reply:
Fixed in [a672afa](https://github.com/bcgov/EPIC.track/pull/2552/commits/a672afa2b9909f75f6483273913a511cd99f590d)